python312Packages.llama-index-core: 0.12.37 -> 0.12.39

fabaff opened the pull request:
Depends on #411394 and #408823

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python312Packages.llama-index-core: 0.12.37 -> 0.12.39

Changelog: https://github.com/run-llama/llama_index/blob/v0.12.39/CHANGELOG.md

fabaff committed:
python312Packages.llama-index-core: update disabled

fabaff committed:
python312Packages.llama-index-indices-managed-llama-cloud: 0.7.1 -> 0.7.2

fabaff committed:
python312Packages.llama-parse: 0.6.23 -> 0.6.25

fabaff committed:
python313Packages.llama-parse: 0.6.25 -> 0.6.33

fabaff committed:
python313Packages.llama-index-vector-stores-chroma: 0.4.1 -> 0.4.2

philiptaron commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 412620`
Commit: `7641667d03f17a2336160b29dfd8d78a3a735941`

---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 141 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>newelle</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.llama-index</summary>
<pre>no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built llama_index-0.12.39-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/llama_index-0.12.39...OK
Repacking wheel as ./llama_index-0.12.39-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for llama_index-0.12.39-py3-none-any.whl
  - llama-index-llms-openai<0.5,>=0.4.0 not satisfied by version 0.3.44</pre>
</details>
</details>

philiptaron reviewed (commented):
`llama-index` is broken by these.

fabaff committed:
python313Packages.llama-index-llms-ollama: 0.5.6 -> 0.6.2

fabaff committed:
python313Packages.llama-cloud: 0.1.23 -> 0.1.26

fabaff committed:
python313Packages.llama-index-core: 0.12.39 -> 0.12.42

Changelog: https://github.com/run-llama/llama_index/blob/v0.12.42/CHANGELOG.md

fabaff committed:
python313Packages.llama-index-cli: 0.4.1 -> 0.4.3

fabaff committed:
python313Packages.llama-index-llms-openai: 0.3.44 -> 0.4.7

fabaff committed:
python313Packages.llama-index-llms-openai-like: 0.3.5 -> 0.4.0

fabaff committed:
python313Packages.llama-index-multi-modal-llms-openai: 0.5.0 -> 0.5.1

fabaff committed:
python313Packages.llama-index-program-openai: 0.3.1 -> 0.3.2

fabaff committed:
python313Packages.llama-index-question-gen-openai: 0.3.0 -> 0.3.1

philiptaron reviewed (dismissed):
Still needs work; `llama-index` still doesn't build.

fabaff committed:
python313Packages.llama-cloud: 0.1.26 -> 0.1.29

fabaff committed:
python313Packages.llama-cloud-services: 0.6.36 -> 0.6.37

Diff: https://github.com/run-llama/llama_cloud_services/compare/refs/tags/v0.6.36...refs/tags/v0.6.37

Changelog: https://github.com/run-llama/llama_cloud_services/releases/tag/v0.6.37

fabaff committed:
python313Packages.llama-index-embeddings-huggingface: 0.5.4 -> 0.5.5

fabaff committed:
python313Packages.llama-index-indices-managed-llama-cloud: 0.7.2 -> 0.7.7

fabaff committed:
python313Packages.llama-parse: 0.6.33 -> 0.6.37

fabaff committed:
python313Packages.llama-index-instrumentation: init at 0.2.0

fabaff committed:
python313Packages.llama-index-workflows: init at 1.0.1

An event-driven, async-first, step-based way to control the
execution flow of AI applications like Agents

fabaff committed:
python312Packages.llama-index-core: 0.12.42 -> 0.12.44

Changelog: https://github.com/run-llama/llama_index/blob/v0.12.44/CHANGELOG.md

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 412620`
Commit: `db694baad5340261396c7b633b3b32d7a47d1c9e`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>newelle</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 156 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.llama-cloud</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-cloud.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-instrumentation</li>
    <li>python312Packages.llama-index-instrumentation.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index-workflows</li>
    <li>python312Packages.llama-index-workflows.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python313Packages.llama-cloud</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-cloud.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-instrumentation</li>
    <li>python313Packages.llama-index-instrumentation.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index-workflows</li>
    <li>python313Packages.llama-index-workflows.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
  </ul>
</details>

fabaff commented:
`lxml-html-clean` is already broken.
